dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -120,6 +122,32 @@ def cleanLength(length):
             raise ValueError("Length must be positive")
         return length
 
+    @staticmethod
+    def cleanRow(row):
+        for key in row.keys():
+            value = row[key]
+            if value is not None:
```

Comment:
Not being a financial expert, I can't rule out ever seeing that.  Checking the field type does seem safer, but there are also a number of fields that are labeled as strings when we know it's really an int (e.g. object class codes).  So if someone gives us a subaccount number of 1,234 or a year of 2,016, do we try to help them?  I can see arguments for both sides here.